python/paddle/base/framework.py

```diff
@@ -231,12 +310,12 @@ def in_pir_mode():
             False
 
             >>> paddle.enable_static()
-            >>> paddle.framework.set_flags({"FLAGS_enable_new_ir_api": True})
-            >>> print(paddle.framework.in_pir_mode())
+            >>> with paddle.new_ir_utils.IrGuard():
+            >>>     print(paddle.framework.in_pir_mode())
```

Comment:
复合代码块用 `...`，参考 [代码示例书写规范](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/dev_guides/style_guide_and_references/code_example_writing_specification_cn.html)
```suggestion
            >>> with paddle.new_ir_utils.IrGuard():
            ...     print(paddle.framework.in_pir_mode())
```

Reply:
谢谢，下个pr提交修改